doc/reference.md

```diff
@@ -807,6 +865,11 @@ errorInfo       | [ErrorInfo](#errorinfo)  | Information about the cause of the
  
  Callback invoked when Buyer Verification flow fails.
 
+---
+ ### CardOnFileBuyerVerificationSuccessCallback
```

Comment:
Looking now, this isn't the right name. Our new flow works for all kinds of cards.

The real issue is that `startCardEntryFlowWithBuyerVerification` uses an `onBuyerVerificationCallback` with `BuyerVerificationDetails`. I believe the original issue is that it takes a `Card` object, which doesn't exist for BuyerVerification.

As such, my proposal will be:

1. Change this to be `BuyerVerificationSuccessCallback`
2. Modify the `BuyerVerificationDetails` object to make the `Card` object optional

If this is not possible, then we should instead call this `onBuyerVerificationFlowSuccessCallback`

Reply:
This works, too.

Reply:
@shivamsaviant Let's go with this approach. Let's make the `Card` object nullable, and keep using `BuyerVerificationSuccessCallback`

Reply:
Okay, I will necessary changes for this suggestion.